citi-onboarding/hacka-pride
# Bugfix/svg animations

## What I did
* Changed website texts and builded to deploy

## How to test
1. Run the server with `nodemon .` and client with `yarn start`
2. Open `localhost:3000` on your browser and check the website
